packages/custom_lint/lib/src/workspace.dart

```diff
@@ -424,6 +424,11 @@ Iterable<String> _findRoots(String path) sync* {
   final directory = Directory(path);
 
   yield* directory.listSync(recursive: true).whereType<File>().where((file) {
+    final relativePathSegments = split(relative(file.path));
```

Comment:
I think the correct fix is to remove the `isChildFromCache` logic and always return `packageConfig.existsSync`

Reply:
This seems to work. Thanks!